python/pyspark/pandas/accessors.py

```diff
@@ -269,20 +269,20 @@ def apply_batch(self, func, args=(), **kwds) -> "DataFrame":
 
```

Comment:
Would you mind fixing chunpp -> chunks too since we're here? see https://github.com/apache/spark/pull/32103/files#diff-eca10d3f01d801e744b5217cb7b571a30ec003e68d1af624086632a90fcc97e2R268

Reply:
https://github.com/apache/spark/pull/32103/files#diff-eca10d3f01d801e744b5217cb7b571a30ec003e68d1af624086632a90fcc97e2R477 this too